### ISSUE TITLE
Seems to break circular dependencies

### ISSUE THREAD

**Issue by ernie** [speedskater/babel-plugin-rewire#65]
I spent hours today troubleshooting an issue in which I had two React components that imported one another. I was using @badsyntax's react-seed project as a starting point for my project, which has the following line:

```javascript
  jsxLoader = ['babel-loader?optional[]=runtime&stage=0&plugins=rewire'];
```

After pulling my hair out as to why one of my modules was seeing the other as undefined, assuming the issue was with my code, I tried removing rewire from the loader configuration, and everything worked as expected.

I'm afraid I don't have much more information to supply at this point, but thought it was worth sharing in case it's a known issue.

**Comment by speedskater** [comment]
@ernie This is the same issue as reported in https://github.com/speedskater/babel-plugin-rewire/issues/59 There is already a fix for this problem 0.1.23-beta-3. At the moment this is only beta version as this was a major change which could result in further errors (which we try to detect and fix ;) ).
Therefore could you please try the beta version?

**Comment by ernie** [comment]
Ah, I didn't see this one when I was looking for a duplicate issue. My apologies.
